api/query/cache/index/index.go

```diff
@@ -244,7 +250,11 @@ func (i *shardedWPTIndex) IngestRun(r shared.TestRun) error {
 		}
 	}
 
-	i.syncStoreRun(r, shardData)
+	defer func() {
```

Comment:
This wasn't deferred previously. Which linter flagged this one? I think you just need to print the warning correct?

Reply:
Also, change the message. It looks like it was copied from above.

Reply:
My bad on this one 😅, I confused it with `syncClearInProgress` above. Fixed!